build.sh

```diff
@@ -5,10 +5,13 @@ set -xeuo pipefail
 # rsync, python2, pygobject3-base are dependencies of ostree-releng-scripts
 # Also add python3 so people can use that too.
 # createrepo_c+yum-utils is used for managing rojig bits.
+# We also install podman+buildah+skopeo to support recursive containerization,
+# and manipulating images.
 dnf -y install rpm-ostree selinux-policy-targeted rpm-build \
     make cargo golang git jq \
     rsync pygobject3-base python3-gobject-base \
-    createrepo_c dnf-utils
+    createrepo_c dnf-utils \
+    podman buildah skopeo
```

Comment:
Hmm, I thought `podman` pulled in `buildah` already, but it looks like that's not in F28 yet.

Reply:
`podman build` works without `buildah` being installed in f28 so it's possible it's not needed.